src/README.md.jinja

```diff
@@ -3,6 +3,7 @@
 {%- endmacro -%}
 {%- if org_slug == "OCA" %}
 [![Runboat](https://img.shields.io/badge/runboat-Try%20me-875A7B.png)](https://runboat.odoo-community.org/builds?repo={{ org_slug }}/{{ repo_slug }}&target_branch={{ odoo_version }})
+[![Find PSC team](https://img.shields.io/badge/PSC-Find%20team-875A7B.png)](https://www.google.com/search?q=site%3Ahttps%3A%2F%2Fodoo%2Dcommunity.org%2Fpsc%2Dteams%20%22{{ repo_slug | urlencode }}%22)
```

Comment:
```suggestion
[![Find PSC team](https://img.shields.io/badge/PSC-Find%20team-875A7B.png)](https://duckduckgo.com/?q=site%3Ahttps%3A%2F%2Fodoo-community.org%2Fpsc-teams+%22{{ repo_slug | urlencode }}%22)
```

this should work @carmenbianca

Reply:
Sadly, not:

![image](https://github.com/OCA/oca-addons-repo-template/assets/973709/5ab9d7a0-a359-4a0e-9725-2a66a114dba8)